Drv/LinuxGpioDriver/CMakeLists.txt

```diff
@@ -11,14 +11,14 @@ if(NOT DEFINED FPRIME_USE_STUBBED_DRIVERS OR FPRIME_USE_STUBBED_DRIVERS)
         "${CMAKE_CURRENT_LIST_DIR}/LinuxGpioDriverComponentImplCommon.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/LinuxGpioDriverComponentImplStub.cpp"
     )
-elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux" OR ${CMAKE_SYSTEM_NAME} STREQUAL "arm-linux-gnueabihf")
+elseif(${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
     set(SOURCE_FILES
         "${CMAKE_CURRENT_LIST_DIR}/LinuxGpioDriverComponentAi.xml"
         "${CMAKE_CURRENT_LIST_DIR}/LinuxGpioDriverComponentImplCommon.cpp"
         "${CMAKE_CURRENT_LIST_DIR}/LinuxGpioDriverComponentImpl.cpp"
     )
 else()
-    message(FATAL_ERROR "Cannot use ${CMAKE_CURRENT_LIST_DIR} with platform ${CMAKE_SYSTEM_NAME}. Consider using -DFPRIME_USE_STUBBED_DRIVERS=ON")
```

Comment:
If you're attempting to compile linux drivers on a non-linux system and don't explicitly note that you want to use stubbed drivers, shouldn't generation error out? For the most part, I think that stubbed drivers should be opt-in, since using them is almost certain going to break the deployment if you try and actually run it. 

Does Ref somehow indirectly wind up pulling in the linux drivers?

Reply:
This is a generation time issue in that these packages are included.  So although Ref does not use these drivers, it still scans them as possibilities in the make system.  Thus any non-Linux build would error on generation. 

With this change you will get the above message at generation time and a "no target found" if you try to use the above items.